src/main/webapp/app/tasks/task-form/task-form.component.ts

```diff
@@ -1,32 +1,41 @@
 import { ChangeDetectionStrategy, Component, EventEmitter, Inject, Output } from '@angular/core';
 import { FormControl, FormGroup, Validators } from '@angular/forms';
 
+// import moment from 'moment';
```

Comment:
those kind of comments - especially in the wrong place - are more confusing than of any help